websocket.go

```diff
@@ -240,13 +253,14 @@ func (s *Stream) reconnect(ctx context.Context, requests []models.WSRequest) (*w
 }
 
 func (s *Stream) subscribe(conn *websocket.Conn, requests []models.WSRequest) error {
+	authorized := false
 	for _, req := range requests {
-		if req.IsPrivateChannel() && !s.authorized {
+		if req.IsPrivateChannel() && !authorized {
 			err := s.auth(conn)
 			if err != nil {
 				return errors.WithStack(err)
 			}
-			s.authorized = true
+			authorized = true
```

Comment:
This looks about right, what error was this encountering before?

Reply:
auth state is not correct after reconnect.